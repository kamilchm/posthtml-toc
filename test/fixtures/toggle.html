<html>
<body>
<h1 id="title1">Title 1</h1>
<p>p1</p>
<h2 id="title2">Title 2</h2>
<p>p2</p>
<h3 id="title3">Title 3</h3>
<p>p3</p>
</body>
</html>
