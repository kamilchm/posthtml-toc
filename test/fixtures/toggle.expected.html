<html>
<body>
<h1 id="title1">Title 1</h1>
<p>p1</p><style>#toctoggle,#toctoggle:checked~ul{display:none}#toctoggle~label:after{content:"hide"}#toctoggle:checked~label:after{content:"show"}#toc label{cursor:pointer}</style><div id="toc"><input type="checkbox" role="button" id="toctoggle"><h2>Test</h2><label for="toctoggle"></label><ul><li><a href="#title2">Title 2</a><ul><li><a href="#title3">Title 3</a></li></ul></li></ul></div>
<h2 id="title2">Title 2</h2>
<p>p2</p>
<h3 id="title3">Title 3</h3>
<p>p3</p>
</body>
</html>
